### PR TITLE
Update docs options command

### PR DESCRIPTION
**Pull request by rohmanhm**
**react-native-cli: 1.2.0** option `--configuration` replaced with `--config`

`react-native run-android --configuration`
replaced with
`react-native run-android --config`

### CODE REVIEW

File: docs/RunningOnDevice.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ $ react-native run-android
 
 > Hint
 >
-> You can also use the `React Native CLI` to generate and run a `Release` build (e.g. `react-native run-android --configuration Release`).
+> You can also use the `React Native CLI` to generate and run a `Release` build (e.g. `react-native run-android --config Release`).
```
Comment (SandroMachado) [inline docs/RunningOnDevice.md]: Why? In the other options we do not use any abbreviation. I don't see any improvements with this change.

You also need to update it [here](https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L245), [here](https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L101) and [here](https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L106).

Comment (rohmanhm) [inline docs/RunningOnDevice.md]:
```
➜  HelloProject react-native run-android --help

  react-native run-android [options]
  builds your app and starts it on a connected Android emulator or device

  Options:

    -h, --help          output usage information
    --install-debug
    --root [string]     Override the root directory for the android build (which contains the android directory)
    --flavor [string]   --flavor has been deprecated. Use --variant instead
    --variant [string]
    --config [string]   Path to the CLI configuration file
```

and when I type `react-native run-android --configuration` show this error

```
➜  HelloProject react-native run-android --configuration

  error: unknown option `--configuration'
```

Here is my versions `react-native-cli`

```
➜  HelloProject react-native --version
react-native-cli: 1.2.0
react-native: 0.38.0
```

**Comment by mkonicek** [comment]
This is not correct, the option is called configuration (and we don't want to change that as that would be a breaking change):

$ react-native run-android --help

  react-native run-android [options]
  builds your app and starts it on a connected Android emulator or device

  Options:

    -h, --help                output usage information
    --root [string]           Override the root directory for the android build (which contains the android directory)
    --flavor [string]         --flavor has been deprecated. Use --variant instead
    --configuration [string]  You can use `Release` or `Debug`. This creates a build based on the selected configuration. If you want to use the `Release` configuration make sure you have the `signingConfig` configured at `app/build.gradle`.
    --variant [string]
    --config [string]         Path to the CLI configuration file

**Comment by rohmanhm** [comment]
But, this is what I see

How I can showing `--configuration` options into my `terminal`?


  ```
$ react-native run-android --help 

  react-native run-android [options]
  builds your app and starts it on a connected Android emulator or device

  Options:

    -h, --help          output usage information
    --install-debug
    --root [string]     Override the root directory for the android build (which contains the android directory)
    --flavor [string]   --flavor has been deprecated. Use --variant instead
    --variant [string]
    --config [string]   Path to the CLI configuration file
```

**Comment by SandroMachado** [comment]
It is not available yet. It is only available in the [0.39.0-rc.0](https://github.com/facebook/react-native/releases/tag/v0.39.0-rc.0).

**Pull request closed without merging**